.rdoc_options

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Can we please limit this file to only what we feel is important?  For example:

```
autolink_excluded_words:
 - Rack
```

Seems like a good idea.  However, I don't think `tab_width: 8` is, as `CHANGELOG.md` seems like the only non-binary file that has tab characters, and even if we did want to treat tabs specially, typical Ruby tab width (indentation level) is 2, not 8.

Reply:
I generated the default file using Doc's own command, and that's what it produced. I am happy to scope this down.